python_ta/__init__.py

```diff
@@ -272,20 +273,13 @@ def reset_linter(config=None, file_linted=None):
         ),
     )
 
+    parent_dir_path = os.path.dirname(__file__)
+    python_ta_dir = os.listdir(parent_dir_path)
+    checkers_index = python_ta_dir.index("checkers")
```

Comment:
This works but is a bit redundant. You don't need to use `checkers_index` at all, since you're just looking for the directory `checkers`, which you know exists. So you can just append `/checkers` directly to the parent directory